src/main/java/inventory/Stock.java

```diff
@@ -40,6 +39,10 @@ public static int getStockCount() {
         return stockCount;
     }
 
+    public static void setStockCount(int stockCount) {
+        Stock.stockCount = stockCount;
+    }
+
```

Comment:
Perhaps can remove setStockCount and corresponding setStockId. Don't think it's logical or safe to include this.